dive_core/data_core.h

```diff
@@ -67,10 +68,10 @@ class DataCore
     CaptureData::LoadResult LoadCaptureData(const char *file_name);
 
     // Parse the capture to generate info that describes the capture
-    bool ParseCaptureData();
+    bool ParseCaptureData(bool is_gfxr_capture = false);
```

Comment:
i feel it is better to not have the default value to force the user know explicitly what value is passed in.

Reply:
Done.